finder.php

```diff
@@ -41,6 +41,6 @@
     //->notPath('wp-includes/theme-compat/footer.php')
     //->notPath('wp-includes/theme-compat/header.php')
     //->notPath('wp-includes/theme-compat/sidebar.php')
-//    ->notPath('wp-includes/Requests/src')
+    //->notPath('wp-includes/Requests/src')
```

Comment:
```suggestion
```

This bad indentation signified a plan to exclude Request v2 from WC stubs.

Reply:
Did you mean the bad indentation should be kept or the whole line should be dropped?

Reply:
Removed! Plan abandoned.

Removed in master.